src/components/base/__tests__/Buttons.test.tsx

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import { render, screen, fireEvent } from "@testing-library/react";
+
+import {
+  Button,
+  SquareSharpButton,
+  LargeButton,
+  SizeInfoButton,
+} from "../buttons";
+
+describe("The component <Button />", () => {
+  const mockButton = {
+    children: "component",
+    disabled: true,
+    color: "red",
+    modifier: "large",
+    sizes: "M",
+  };
+  it("Should render Button component with children", async () => {
+    render(<Button>{mockButton.children}</Button>);
```

Comment:
Esta renderizando um componente com um objeto dentro, não seria um destructuring no componente? 
```suggestion
    render(<Button {...mockButton}>Click Me</Button>);
```